### PR TITLE
Feature/summary

### PR DESCRIPTION
**Pull request by nroode**

### CODE REVIEW

File: client/src/App.jsx [hunk 1]
```diff
@@ -1,5 +1,8 @@
 import React, { useState } from 'react';
 
+
+import FullView from './components/FullView';
+import Summary from './components/Summary';
```
Comment (terrancexin) [inline client/src/App.jsx]: 😃

File: client/src/App.jsx [hunk 2]
```diff
@@ -12,7 +15,7 @@ export default function App() {
   return (
     <FullAppContext.Provider value={{ currentUser, setCurrentUser, setAuth, auth }}>
       <Header />
-      <div className="full-app">{auth ? <AppList /> : <Login />}</div>
+      <div className="full-app">{auth ? <FullView /> : <Login />}</div>
```
Comment (terrancexin) [inline client/src/App.jsx:16]: Why the changes to `FullView` here?

File: client/src/Components/Header/index.jsx [hunk 3]
```diff
@@ -8,13 +8,15 @@ export default function Header() {
 
   return (
     <header>
-      <h1 className="header">Job Tracker</h1>
-      {currentUser && <h1>{currentUser.username}</h1>}
+	  <h1 className="header">Job Tracker</h1>
+	  <div>
+      {currentUser && <div className="user"><i className="fas fa-user"></i><h3 className="username">{currentUser.username}</h3></div>}
```
Comment (terrancexin) [inline client/src/Components/Header/index.jsx]: empty tags can be self closing `<i />`

File: client/src/components/Summary/Summary.scss [hunk 4]
```diff
@@ -0,0 +1,33 @@
+@import './../../variables.scss';
+
+// .dev-img-container {
+//     width: 40%;
+// }
```
Comment (terrancexin) [inline client/src/components/Summary/Summary.scss]: remove comments

File: client/src/components/Summary/Summary.scss [hunk 5]
```diff
@@ -0,0 +1,33 @@
...
+.dev-img {
+width: 100%;
+padding: 10px;
+}
+
+.summary-container {
+    background: $white;
+    height: 100%;
+    max-width: 30%;
+    // flex-grow: 1;
```
Comment (terrancexin) [inline client/src/components/Summary/Summary.scss]: remove comments

File: client/src/components/Summary/index.jsx [hunk 6]
```diff
@@ -0,0 +1,21 @@
+import React, { useState } from 'react';
```
Comment (terrancexin) [inline client/src/components/Summary/index.jsx]: remove unused `useState`

File: client/src/App.jsx (hunk 2 above)
Comment (nroode) [reply client/src/App.jsx:16]: We don't want to look at just the list of job applications.  This includes the Summary too.

**Review by clockwerkz — APPROVED** [review]
I don't think the unused lines will have much of an impact. Up to you to remove them; otherwise we can do a pass after we get the work done.

**Review by paul1124 — APPROVED** [review]
